ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js

```diff
@@ -211,16 +219,14 @@ describe('component replace - Authoring', function () {
             cy.openAuthoring("/conf/core-components-examples/settings/wcm/templates/af-blank-v2/structure");
             updateAllowedComponent(true);
 
-            cy.openSiteAuthoring(replaceCompPageUrl);
+            cy.openSiteAuthoring(replaceCompPagePath);
```

Comment:
how is this different in cloud? I mean what is url in cloud vs 6.5

Reply:
This is not different. We should change the variable name in master also as both variables have same value.

Reply:
If value is same - why was it failing earlier?